### ISSUE TITLE
additional use case - content author/designer ?

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-easy-checks#128]
Jade drafted:
> Content author/designer —  Tim produces online content for a variety of clients. He uses the quick check resource as a way to ensure his content has basic accessibility, and also to demonstrate to clients that he's considered accessibility in his approach.

From [10 March minutes](https://www.w3.org/2023/03/10-eo-minutes#item02) include comments such as:
> If you're a freelancer and need to show a client what you can do, they may want to use this. They may want to have accessibility represented in their work without being able to get fully trained up on the topic.
...
> Coders may have more awareness but designers and writers have been skipped in the resources. That's getting better but they don't have a lot.
...
> We can add "He started using..." to make it more clear.

Thoughts on this as a [use case](https://www.w3.org/WAI/EO/wiki/Easy_Checks_2022#Use_Cases) for Easy Checks revision?

**Comment by shawna-slh** [comment]
I'm not sure that we want this to be a target use case, as least as written. Sure someone could use Easy Checks as one learning point. However, I think we would want this persona to learn more &mdash; take the [course](https://www.w3.org/WAI/courses/foundations-course/) and/or read the [fundamentals resources](https://www.w3.org/WAI/fundamentals/), [tips for designing](https://www.w3.org/WAI/tips/designing/), [tips for writing](https://www.w3.org/WAI/tips/writing/), [clear content](https://www.w3.org/WAI/WCAG2/supplemental/objectives/o3-clear-content/).

And any demonstration to clients should be broader than Easy Checks I think.

<hr>

Is it useful to add a use case with this persona and write it that Easy Checks is just one of several resources they use to learn accessibility? Or, do we not need to bother adding this use case?

**Comment by jasperjeurens** [comment]
I think these are all valid points. I would highly advocate towards making the position of the Easy Checks page within the larger accessibility ecosystem/resources very clear.

**Comment by beltonTPG** [comment]
I think that I agree with @shawna-slh that this use case doesn't necessarily apply here. The target audience are those that don't deal in accessibility as part of their everyday, but rather use Easy Checks to do quick, high-level evaluations for the purpose of moving to a next step (essentially). Content creators should be going beyond the quick and easy and have more in-depth accessibility knowledge.

>  and also to demonstrate to clients that he's considered accessibility in his approach.

I think the client is more of a use case to check that the content creator has considered accessibility.

**Comment by iadawn** [comment]
Agree with @MicheleAWilliams-A11y's comment. However, I don't know how well that group is supported in general. Might be a resource gap!
